src/tables/math.rs

```diff
@@ -0,0 +1,1062 @@
+//! A [MATH Table](https://docs.microsoft.com/en-us/typography/opentype/spec/math) implementation.
+
+use crate::GlyphId;
+use crate::gpos::Device;
+use crate::opentype_layout::Coverage;
+use crate::parser::{
+    Offset16, Stream,
+    FromSlice, FromData, Offset,
+    LazyArray16, LazyOffsetArray16,
+};
+
+/// A [MATH Table](https://docs.microsoft.com/en-us/typography/opentype/spec/math).
+#[allow(missing_debug_implementations)]
+#[allow(missing_docs)]
+#[derive(Clone)]
+pub struct Table<'a> {
+    pub major_version: u8,
```

Comment:
I don't think we need versions. All version-specific nuances must be handled by the library implicitly.

Reply:
Handled by removing the version fields, and comparing against `[1, 0]` during parsing. (Currently, there is only version 1.0.)